packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -178,6 +178,7 @@ Enumerates Groups in your organization with pagination. A subset of Groups can b
 | filter    | [Filter expression](/docs/reference/api-overview/#filtering) for Groups      | Query     | String   | FALSE    |         |
 | limit     | Specifies the number of Group results in a page                                            | Query     | Number   | FALSE    | 10000   |
 | q         | Finds a group that matches the `name` property                                               | Query     | String   | FALSE    |         |
+| expand        | If specified, it causes additional metadata to be included in the response. Possible values are `stats` or `app`.                                             | Query     | String   | FALSE    |         |
```

Comment:
Note that it's also valid to use both values together, ie. `expand=stats,app`. But this description looks good to me.